src/content/docs/en/chromebook/faq-and-troubleshooting.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Troubleshooting and FAQs
+description: "Troubleshooting tips and tricks, and FAQs"
+---
+
+import Alert from "../../../../components/Docs/Alert.astro";
+
+## Troubleshooting
+
+#### Chromebook fails to boot to internal disk
+
+Assuming you are selecting 'Boot to Internal Disk', or pressing <kbd>ctrl</kbd>+<kbd>D</kbd>> does not fully boot, this usually means the device has died. Plug in your Chromebook. If the issue continues to persist, hop into [our chats](/en/community/community) and make a support forum post.
+
+#### Keyboard is not properly mapped
+
+Try running Weirdtreething's keyboard [remap script](https://github.com/WeirdTreeThing/cros-keyboard-map) (creates a Keyd config)
+
+#### Chromebook audio doesn't work
+
+Try running Weirdtreething's [audio script](https://github.com/WeirdTreeThing/chromebook-linux-audio) manually.
```

Comment:
covered by chromebook kernel and not needed

Reply:
Changed copy a bit, but covering all ground.

Also provides link if anyone wants to kill their speakers.

Reply:
great